Tests/FluentAssertions.Specs/Numeric/NumericDifferenceAssertionsSpecs.cs

```diff
@@ -0,0 +1,703 @@
+using System;
+using Xunit;
+using Xunit.Sdk;
+
+namespace FluentAssertions.Specs.Numeric
+{
+    public class NumericDifferenceAssertionsSpecs
+    {
+        public class Be
+        {
+            [Theory]
+            [InlineData(8, 5)]
+            [InlineData(1, 9)]
+            public void When_an_int_difference_between_small_numbers_it_should_not_add_difference_message(int value, int expected)
```

Comment:
We're trying to move away from the "When_abc_it_should_def" pattern heavily used in these tests as we realized it sometimes led to slightly awkward test names.
The new "pattern" (or lack of) is still about focusing of the behavior and not the implementation details, but relaxing/removing the requirement about having to fit into a rigid naming scheme.
See https://enterprisecraftsmanship.com/posts/you-naming-tests-wrong/ for more details.
 
```suggestion
            public void The_difference_between_small_ints_is_not_included_in_the_message(int value, int expected)
```